CRM/Profile/Page/Router.php

```diff
@@ -97,11 +97,12 @@ public function run($args = NULL) {
     }
 
     if ($secondArg == 'edit' || $secondArg == 'create') {
+      $allowRemoteSubmit = Civi::settings()->get('remote_profile_submissions');
```

Comment:
@totten @xurizaemon The way this was before hiding underneath the If in the next line, i'm sure would have led to a unknown variable $allowRemoteSubmit in the create action